schemas/trademark-schema.json

```diff
@@ -0,0 +1,194 @@
+      "items": {
+        "$ref": "includes/entity.json"
+      }
+    },
+    "agent": {
+      "$ref": "includes/entity.json",
+      "description": "Details of the agent"
+    },
+    "representative": {
+      "$ref": "includes/entity.json",
+      "description": "Details of the representative"
+    },
+    "mark_details": {
+      "description": "Details of the trademark itself (rather than the registration)",
+      "type": "object",
+      "properties": {
+        "mark_text": {
+          "description": "Text of the trademark",
+          "type": "string"
+        },
+        "mark_category": {
+          "description": "Category of mark",
+          "type": "string",
+          "enum": [
+            "text",
+            "image",
+            "sensory_mark"
+          ]
+        },
+        "mark_type": {
+          "description": "Free text description of type mark (as provided by register)",
+          "type": "string"
+        },
+        "mark_image": {
+          "image_uid": {
+            "description": "uid of image",
+            "type": "string"
+          },
+          "image_format": {
+            "description": "Type of image format",
+            "type": "string"
+          },
+          "image_url": {
+            "description": "URL of image",
+            "type": "string"
+          },
+          "image_description": {
+            "description": "Description of image in words",
+            "type": "string"
+          }
+        },
+        "classification": {
+          "type": "array",
+          "description": "Classification of the trademark",
+          "items": {
+            "oneOf": [{
+                "type": [
+                  "string"
+                ],
+                "minLength": 1
+              },
+              {
+                "$ref": "#/definitions/nice_classification"
+              },
+              {
+                "$ref": "includes/classification.json"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "uid": {
+      "description": "Unique id/reference for the trademark registration",
+      "type": "string"
+    },
+    "source_url": {
+      "description": "Url of the source of the data (e.g. download URL), or if there is not persistent URL the page from which it can be found (e.g. search page). This field is required.",
+      "type": "string",
+      "minLength": 11
+    },
+    "confidence": {
+      "description": "Confidence in accuracy of data",
+      "type": "string",
+      "enum": [
+        "HIGH",
+        "MEDIUM",
+        "LOW"
+      ]
+    },
+    "registration_date": {
+      "description": "Date on which the mark was registered",
+      "type": "string",
+      "format": "date"
+    },
+    "expiry_date": {
+      "description": "Date on which the registration ended, or is scheduled to expire",
+      "type": "string",
+      "format": "date"
+    },
+    "start_date": {
+      "description": "Date on which the mark was registered",
```

Comment:
@CountCulture @thomasmarshall Do you think now we should just say something to the effect of "This is copied from the `registration_date`" (and ditto for `end_date` and `expiry_date`)? Otherwise it looks a bit weird.